### PR TITLE
Removed DNS request to get app service name and added sleep before swagger import

### PR DESCRIPTION
**Pull request by jack-education**
Removed DNS request to get app service name and added sleep before swagger import

The sleep is needed so the whitelisted IP is in effect.

### CODE REVIEW

File: Infrastructure-Scripts/Import-ApimSwaggerApiDefinition.ps1 [hunk 1]
```diff
@@ -76,6 +76,7 @@ function Add-AppServiceWhitelist ($AppServiceResourceGroup, $AppServiceName) {
         Write-Output "Whitelisting $MyIp"
         $Priority = ($IpRestrictions.MainSiteAccessRestrictions | Where-Object { $_.Action -eq "Allow" }).Priority[-1] + 1
         $null = Add-AzWebAppAccessRestrictionRule -ResourceGroupName $AppServiceResourceGroup -WebAppName $AppServiceName -Name "DeployServer" -IpAddress "$MyIp/32" -Priority $Priority -Action Allow
+        Start-Sleep -Seconds 5
```
Comment (nbowes24) [inline Infrastructure-Scripts/Import-ApimSwaggerApiDefinition.ps1:79]: Could this be -AsJob and then a Wait-Job instead of sleep?

Comment (jack-education) [reply Infrastructure-Scripts/Import-ApimSwaggerApiDefinition.ps1:79]: The Add-AzWebAppAccessRestrictionRule cmdlet completes, the sleep is needed so the whitelisted IP is in effect.

Added to PR description

File: Infrastructure-Scripts/Import-ApimSwaggerApiDefinition.ps1 [hunk 2]
```diff
@@ -64,8 +64,8 @@ function Get-AllSwaggerFilePaths ($SwaggerHtml) {
 }
 
 function Get-AppServiceName ($ApiBaseUrl){
-    $CustomHostname = $ApiBaseUrl -replace ".*https://"
-    $AppServiceName = (Resolve-DnsName -Name $CustomHostname)[0].NameHost.split('.')[0]
+    $AppServices = Get-AzWebapp -ResourceGroupName das-test-apimendp-rg
```
Comment (adam230594) [inline Infrastructure-Scripts/Import-ApimSwaggerApiDefinition.ps1:67]: hardcoded?

**Comment by adam230594** [comment]
Can we move the Start-Sleep logic into the Read-SwaggerHtml Function and have some error catching in there for flexibility?

**Pull request merged**